posthog/models/event_definition.py

```diff
@@ -19,6 +19,7 @@ class EventDefinition(UUIDModel):
     query_usage_30_day: models.IntegerField = models.IntegerField(
         default=None, null=True,
     )  # Number of times the event has been used in a query in the last 30 rolling days (computed asynchronously)
+    verified: models.BooleanField = models.BooleanField(default=False, null=False)
```

Comment:
I wonder if we should consider something like `verified_at` & `verified_by` instead of just a `boolean`, seems like very valuable information to show on the UI.

Reply:
that's a good idea. Agree w/ verified_by but I think verified_at would become repetitive metadata if we implement some kind of general insight history